### PR TITLE
feat: add card component

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added text-overlay card

### CODE REVIEW

**Comment by iamalhera** [comment]
looks good, just add one more card or convert one for text-overlay

File: component/badge/badge.css [hunk 1]
```diff
@@ -27,11 +27,43 @@
   background-color: var(--default-badge-color);
 }
 
+.badge-text {
+  width: max-content;
+  height: fit-content;
+  padding: 4px 6px;
```
Comment (AshutoshM29) [inline component/badge/badge.css]: You can use rem instead of px!
other than that looks nice
you can merge it🚀

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]

File: index.css [hunk 2]
```diff
@@ -54,3 +56,32 @@ a {
...
 }
+
+/* utility classes*/
+
+.d-flex {
+  display: flex;
+}
+
+.margin-r-5 {
+  margin-right: 5px;
```
Comment (AshutoshM29) [inline index.css:68]: Again try using rem instead of px

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]

File: component/badge/badge.css (hunk 1 above)
Comment (Sahil25061999) [reply component/badge/badge.css]: done

File: component/card/card.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+.card-head,
+.card-foot,
+.card-content {
+  padding: 0 1rem;
+  margin-bottom: 10px;
+}
+
+.card-subheading {
+  font-size: 14px;
```
Comment (rahulk31) [inline component/card/card.css:53]: plan to make use of relative values.

**Review by rahulk31 — APPROVED** [review]
UI is great. Code looks good to me. Clean and readable 👍

**Comment by kgajriaAgarwal** [comment]
The Ui is good.Although it has 1 problem on adding extra text the height of the card is changing  and this might create a problem in ecommerce. so you can use text-overflow: ellipsis; property to avoid the height manipulation.

File: component/badge/badge.css [hunk 4]
```diff
@@ -21,17 +21,51 @@
   right: -10%;
   top: -30%;
   color: white;
```
Comment (deekshithmd) [inline component/badge/badge.css]: using color variables will be good

File: component/badge/badge.css [hunk 5]
```diff
@@ -21,17 +21,51 @@
...
-  font-size: 14px;
```
Comment (deekshithmd) [inline component/badge/badge.css:24]: using rem instead of px will be good

File: component/card/card.html [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+          <button class="btn btn-only-icon">
+            <span class="fas fa-heart"></span>
+          </button>
+        </div>
+      </div>
+      <div class="card">
+        <button class="btn btn-float close">
+          <span class="fa-solid fa-xmark"></span>
+        </button>
+        <div class="image-container">
```
Comment (deekshithmd) [inline component/card/card.html:61]: try to avoid using lot of div tag

File: component/badge/badge.css (hunk 4 above)
Comment (Sahil25061999) [reply component/badge/badge.css]: done

**Pull request merged**